docs/src/data/hid.js

```diff
@@ -5184,7 +5184,7 @@ export default [
     os: {
       windows: null,
       linux: true,
-      android: null,
+      android: false,
```

Comment:
Should we seek a second verification for this one before listing `C_BRIGHTNESS_DEC` as unsupported on Android?

It seems strange that `C_BRIGHTNESS_INC` does work but not `C_BRIGHTNESS_DEC`.

I noticed in https://github.com/zmkfirmware/zmk/pull/600#discussion_r776129455 that `C_BRIGHTNESS_DEC` didn't work for Windows during initial testing a year ago, but that it did work in subsequent testing more recently. (So perhaps something changed under the hood that may have resolved this?)

Reply:
It also seems reasonable to me to have someone check this. If someone wants to test it and confirm this functionality, I'll commit a suggested change.

Reply:
FWIW, on Android 10 neither `C_BRIGHTNESS_INC` nor `C_BRIGHTNESS_DEC` work for me.